content/docs/gui/customizing/_index.md

```diff
@@ -0,0 +1,226 @@
+[artifacts]({{< ref "/docs/artifacts/" >}}).
+These artifacts contain templates written in
+[Go's templating language](https://pkg.go.dev/text/template)
+that then produce the HTML that the user ultimately sees and interacts with.
+
+The artifact specifies a template containing markdown plus elements written in
+the Go Template Language. This allows you to also include VQL queries
+(server-side only, since this is where the templates are run!) in your template,
+and display the results in tabular or various graphical formats.
+
+{{% notice info "Some web development experience is required" %}}
+
+Customizing GUI content does require a reasonable degree of understanding of
+HTML as well as of Go's templating language. Typically customization amounts to
+small tweaks where you might make a change to the layout or customize specific
+elements that you don't like, although extensive customization can be done if
+you really need to.
+
+There are many excellent guides to templating and HTML available on the internet
+and for that reason this is not intended to be a tutorial on those topics.
+
+If this is all new to you, you can still experiment by changing one small thing
+at a time. Because the customization is done via a custom artifact, if you mess
+it up you can always delete the custom artifact and Velociraptor will then
+revert to using the default built-in one!
+
+{{% /notice %}}
+
+## Which pages can be customized?
+
+Currently there are 3 pages in the GUI that support full customization.
+
+These, and their corresponding artifacts, are:
+
+- `Server.Internal.Welcome` - The Welcome page
+  ![](welcome.png)
+
+- `Server.Monitor.Health` - The Server Dashboard page
+  ![](dashboard.png)
+
+- `Generic.Client.Info` - The VQL Drilldown page which shows data about any
+  selected client.
+  ![](vql_drilldown.png)
+
+#### Including image content
+
+The GUI is not served from disk which means that, by default, you cannot place
+images on disk and expect them to be picked up and served like a typical web
+server. Images that the GUI uses are compiled into the binary as static web
+assets and then served from memory.
+
+So what do you do if you want to include custom images? Well there are a few
+options to consider:
+
+1. It is possible to include links to images from external locations using the
+   `<img>` HTML tag, although this may be problematic if there is proxying or
+   content filtering in your network environment, or if the external image
+   hosting is unreliable or slow.
+
+2. If you are doing extensive customization or customization that requires
+   compiling your own server binary then these images can be included in the
+   source tree under `gui/velociraptor/src/public` which will then serve these
+   as static assets.
+
+3. The config allows for serving static files from a disk location using the
+   [GUI.reverse_proxy.url]({{< ref "/docs/deployment/references/#GUI.reverse_proxy.url" >}})
+   setting.
+
```

Comment:
We probably need a bit more detail here:

Sample config
```yaml
GUI:
   reverse_proxy:
       - route: /images/
         url: file:///opt/www/
         require_auth: true 
```

Then the image at /opt/www/foo.png will be available using href="/images/foo.png"